po/ca.po

```diff
@@ -7,82 +7,83 @@ msgstr ""
 
 #: src/MainWindow.vala:76
 msgid "Calculating checksum…"
-msgstr ""
+msgstr "Calculant checksum…"
 
 #: src/MainWindow.vala:110 data/com.github.artemanufrij.hashit.desktop.in:3
 #: data/com.github.artemanufrij.hashit.desktop.in:4
 #: data/com.github.artemanufrij.hashit.appdata.xml.in:7
 msgid "Hash It"
-msgstr ""
+msgstr "Hash It"
 
 #: src/MainWindow.vala:115
 msgid "Open a file"
-msgstr ""
+msgstr "Obrir un fitxer"
 
 #: src/MainWindow.vala:126
 msgid "Choose an algorithm"
-msgstr ""
+msgstr "Escollir un algorisme"
 
 #: src/MainWindow.vala:139
 msgid "(optional) paste a reference hash here…"
-msgstr ""
+msgstr "(opcional) enganxar un hash de referència aquí…"
 
 #: src/MainWindow.vala:144
 msgid "Choose a file or drag one onto this window…"
-msgstr ""
+msgstr "Escollir un fitxer o arrosegar un en aquesta finestra…"
 
 #: src/MainWindow.vala:153
 msgid "Copy result…"
-msgstr ""
+msgstr "copiar resultat…"
 
 #: src/MainWindow.vala:201
 msgid "Open"
-msgstr ""
+msgstr "Obrir"
 
 #: src/MainWindow.vala:203
 msgid "_Cancel"
-msgstr ""
+msgstr "_Cancel·lar"
 
 #: src/MainWindow.vala:204
 msgid "_Open"
-msgstr ""
+msgstr "_Obrir"
 
 #: src/MainWindow.vala:207
 msgid "All files"
-msgstr ""
+msgstr "Tots els fitxers"
 
 #: data/com.github.artemanufrij.hashit.desktop.in:5
 #: data/com.github.artemanufrij.hashit.appdata.xml.in:8
 msgid "Generate and verify file hashes"
-msgstr ""
+msgstr "Generar i verificar el hash del fitxer"
 
 #: data/com.github.artemanufrij.hashit.desktop.in:6
 msgid "hash;checksum;md5;sha256;"
-msgstr ""
+msgstr "hash;checksum;md5;sha256;"
 
 #: data/com.github.artemanufrij.hashit.desktop.in:8
 msgid "com.github.artemanufrij.hashit"
-msgstr ""
+msgstr "com.github.artemanufrij.hashit"
 
 #: data/com.github.artemanufrij.hashit.appdata.xml.in:10
 msgid ""
 "A tool that allows you to ensure a file hasn't become corrupted by comparing "
 "its hashsum"
-msgstr ""
+msgstr "Una ferramenta que et permet assegurar que un fitxer no està corrupte comparant "
+"el seu hashsum"
```

Comment:
Instead of a possessive (“el seu”) use the pronom feble _ne_: “**comparant-ne** la suma de comprovació”